var/spack/repos/builtin/packages/scorep/package.py

```diff
@@ -90,7 +90,8 @@ def url_for_version(self, version):
     # two components of cube -- cubew and cubelib.
 
     # SCOREP 8
-    depends_on("binutils", type="link", when="@8:")
+    depends_on("binutils libs=shared", type="link", when="@8:")
+    conflicts("binutils libs=static")  # cap this if and when we find a reliable workaround
```

Comment:
`spack audit` is complaining that `scorep` has no `libs` variants. Can you try:
```suggestion
    conflicts("^binutils libs=static")  # cap this if and when we find a reliable workaround
```
and check if this solves the issue?